methods/getCacheKey.js

```diff
@@ -1,3 +1,16 @@
-module.exports = function(url) {
-  return `${this.settings.app.redis.prefix}-${url}`;
+const url = require('url');
+const qp = require('query-parse');
+
+module.exports = function(path) {
```

Comment:
Maybe pass the url object that you already have with path and query already separated